src/modules/auth/controller.js

```diff
@@ -28,6 +31,34 @@ const tryAuth = async (req, res, next) => {
   }
 };
 
+const refreshToken = async (req, res, next) => {
+  try {
+    const user = userService.getUser({ id: req.user.id });
+
+    if (!user) {
+      next(new UnauthorizedError(INVALID_TOKEN_MESSAGE));
```

Comment:
`return` statement
By the way in this case it's critical - without `return` statement you can get an error like `Can't send headers after they are sent`